src/Repository/VersionRepository.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Repository;
+
+use Buddy\Repman\Entity\Organization\Package\Version;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * @method Version|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Version|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Version[]    findAll()
+ * @method Version[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ * @extends ServiceEntityRepository<Version>
+ */
+class VersionRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, Version::class);
+    }
+
+    public function remove(UuidInterface $id): void
+    {
+        $version = $this->find($id);
```

Comment:
to prevent unnecessary hydration you can just write a plain sql with `DELETE` statement